active/NavigationView/HierarchicalNav/HierarchicalNavigationView.md

```diff
@@ -0,0 +1,363 @@
+                            new Category() { 
+                                Name  = "Menu Item 2", 
+                                Icon = "Icon",
+                                Children = new ObservableCollection<Category>() {
+                                    new Category() { Name  = "Menu Item 3", Icon = "Icon" },
+                                    new Category() { Name  = "Menu Item 4", Icon = "Icon" }
+                                }
+                            }
+                        }
+                    }
+                }
+            },
+            new Category(){
+                Name = "Menu Item 5",
+                Icon = "Icon",
+                Children = new ObservableCollection<Category>() {
+                    new Category(){
+                        Name = "Menu Item 6",
+                        Icon = "Icon",
+                        Children = new ObservableCollection<Category>() {
+                            new Category() { Name  = "Menu Item 7", Icon = "Icon" },
+                            new Category() { Name  = "Menu Item 8", Icon = "Icon" }
+                        }
+                    }
+                }
+            },
+            new Category(){ Name = "Menu Item 9", Icon = "Icon" }
+        };
+    }
+
+    private void OnItemInvoked(object sender, NavigationViewItemInvokedEventArgs e)
+    {
+        var clickedItem = e.InvokedItem;
+        var clickedItemContainer = e.InvokedItemContainer;
+    }
+
+    private void OnItemExpanding(object sender, NavigationViewItemExpandingEventArgs e)
+    {
+        var nvib = e.ExpandingItemContainer;
+        if(nvib != null)
+        {
+            var name = "Last Expanding: " + nvib.Content;
+            ExpandingItemLabel.Text = name;
+        }
+        else
+        {
+            ExpandingItemLabel.Text = "Last Expanding: ERROR - No container returned!";
+        }
+    }
+
+    private void OnItemCollapsed(object sender, NavigationViewCollapsedEventArgs e)
+    {
+        var nvib = e.CollapsedItemContainer;
+        if (nvib != null)
+        {
+            var name = "Last Collapsed: " + nvib.Content;
+            CollapsedItemLabel.Text = name;
+        }
+        else
+        {
+            CollapsedItemLabel.Text = "Last Collapsed: ERROR - No container returned!";
+        }
+    }
+}
+```
+
+## Selection
+By default, any item can contain children, be invoked, or be selected.
+There's more detail about the [ItemInvoked](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.navigationview.iteminvoked?view=winui-2.3) and [SelectionChanged](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.navigationview.selectionchanged?view=winui-2.3) events later in this document.
+When providing users with a hierarchical tree of navigation options, you may choose to make parent items non-selectable, for example when your app doesn't have a destination page associated with that parent item.
```

Comment:
But it can still be invoked, so what am I supposed to do when invoked? Just ignore the invoke? Do I have to create a custom automation peer that does not support the invoke pattern?

Reply:
Discussion: 
Note that these APIs exist -- Invoked and SelectsOnInvoked -- but the point here is that regardless of how this property is set, it will expand/collapse. So really this property just got a lot more useful.